samples/bluetooth/mesh/light/README.rst

```diff
@@ -5,25 +5,29 @@ Bluetooth: Mesh Light
 
 The Bluetooth Mesh Light sample demonstrates how to set up a basic Mesh server model application and control LEDs with the Bluetooth Mesh, using the :ref:`bt_mesh_onoff_readme`.
 
+This sample is required when working with the :ref:`bluetooth_mesh_light_switch` sample.
```

Comment:
@trond-snekvik , I'm a bit confused about the relationship with the light switch sample.
If the switch sample is used, does the testing procedure for mesh light changes?

Reply:
The test procedure for the light is the same. The last step (interacting with the model) can be skipped, but I don't think it needs to be addressed. Currently the mesh light sample is completely self contained, while the light switch optionally complements it. I think this line is sufficient